src/Service/Generator/DocBlock/Custom/Method.php

```diff
@@ -9,7 +9,7 @@
 
 class Method extends BaseTag
 {
-    protected $name = 'method';
+    protected string $name = 'method';
```

Comment:
No idea why this would help but looks good to me if this works.

Reply:
https://github.com/phpDocumentor/ReflectionDocBlock/commit/c6d49fc451c8e04501650c67474df943cbbb5547

This commit added a type hint to the BaseTag class.